components/03-modules/gallery/gallery.hbs

```diff
@@ -0,0 +1,86 @@
+<div class="
+         gallery {{ class }}
+         {{#if typeHorizontal }}
+             gallery--horizontal
+         {{else}}
+             gallery--vertical
+         {{/if}}
+     "
+     {{{ attributes }}}
+>
+    <div class="
+             gallery__stage
+             {{ stage.class }}
+             {{#if typeHorizontal }}
+                 gallery__stage--horizontal
+             {{else}}
+                 gallery__stage--vertical
+             {{/if}}
+         "
+    >
+        <div class="gallery__arr gallery__arr--prev">
```

Comment:
this div is necessary here? I don't see any styles to `arr` class

Reply:
yes, removed it as it's not in alpaca design (it was just initially created to match more or less magento fotorama)